test/functional/lib/tests.cpp

```diff
@@ -1675,12 +1675,19 @@ WaitForWfpQuarantine(
     _In_ const TestInterface& If
     );
 
+#pragma warning(push)
+#pragma warning(disable:6101) // We don't set AckNum for UDP
 static
 wil::unique_socket
-CreateUdpSocket(
+CreateTcpUdpSocket(
```

Comment:
Can you create two separate top-level functions to create UDP and TCP sockets? IMO we shouldn't have to pass a bunch of TCP-related parameters (or NULL, TRUE, etc.) to create a plain UDP socket.